neo4j/src/test/scala/magnolify/neo4j/ValueTypeSuite.scala

```diff
@@ -76,4 +79,17 @@ class ValueTypeSuite extends MagnolifySuite {
     assert(!fields.map(record.get).exists(_.isNull))
     assert(!record.get("INNERFIELD").get("INNERFIRST").isNull)
   }
+
+  test("AnyVal") {
+    val vt: ValueType[HasValueClass] = implicitly
```

Comment:
Let's use the instance for the `test` too and trigger the macro with the `apply`
```suggestion
    implicit val vt: ValueType[HasValueClass] = ValueType[HasValueClass]
```